pkg/cmd/config/config.go

```diff
@@ -0,0 +1,144 @@
+package config
+
+import (
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/AlecAivazis/survey/v2"
+	cfg "github.com/GetStream/stream-cli/pkg/config"
+	"github.com/cheynewallace/tabby"
+	"github.com/spf13/cobra"
+)
+
+func NewRootConfigCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config",
+		Short: "Manage app configurations",
+	}
+
+	cmd.AddCommand(newAppCmd(), removeAppCmd(), listAppsCmd(), setAppDefaultCmd())
+
+	return cmd
+}
+
+func newAppCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "new",
+		Short: "Add a new application",
+		Long:  "Add a new application which can be used for further operations",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runQuestionnaire(cmd)
+		},
+	}
+}
+
+func removeAppCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "remove [app-name]",
+		Short: "Remove an application. This operation is irrevocable",
+		Args:  cobra.ExactArgs(1),
```

Comment:
Suggestion as a nice to have: we can support removing more than one app in one go

Reply:
✅